modules/ROOT/pages/to-find-info.adoc

```diff
@@ -327,7 +327,7 @@ This API will be found by these searches:
 |leagueid |URI parameter name
 |team |Type name
 |leagues leagueId teams |Endpoint path
-|`/leagues/{leagueId}/teams` |Endpoint path
+|`/leagues/\{leagueId\}/teams` |Endpoint path
```

Comment:
Unless the { } symbols are actually in the URL, it's easier to use something like:
`/leagues/LEAGUE_ID/teams` 
So that you aren't bound to try to escape characters.

Reply:
True, that's clear enough. Fixed to use this.